Formula/premake4.rb

```diff
@@ -0,0 +1,46 @@
+class Premake4 < Formula
+  desc "Write once, build anywhere Lua-based build system"
+  homepage "https://premake.github.io/"
+  url "https://downloads.sourceforge.net/project/premake/Premake/4.4/premake-4.4-beta5-src.zip"
+  sha256 "0fa1ed02c5229d931e87995123cdb11d44fcc8bd99bba8e8bb1bbc0aaa798161"
+  version_scheme 1
```

Comment:
This is leftover from the previous `premake` formula. Should I remove it? 🤔

Reply:
Also, I've removed all bottles so they get regenerated, is that ok?

Reply:
No, you shouldn't touch the bottle block. And version_scheme can be removed.

Reply:
Ok, I'll bring bottles back and remove version scheme 👍